spec/api/share_spec.rb

```diff
@@ -1,18 +1,164 @@
+            email_valid: true,
+            self_referral: false,
+            unsubscribed: false,
+            blacklisted: false,
+            meets_criteria: true,
+            sharable: true
+          },
+          "friend2@example.com": {
+            currently_sent: true,
+            previously_sent: false,
+            email_valid: true,
+            self_referral: false,
+            unsubscribed: false,
+            blacklisted: false,
+            meets_criteria: true,
+            sharable: true
+          }
+        },
+        reward: nil,
+        stats: {
+          currently_sent: 2,
+          currently_not_sent: 0,
+          previously_sent: 0,
+          total_sent: 2,
+          sent_limit_exceeded: false,
+          left_emails: 20
+        },
+        success: true,
+        validation_only: false
+      }
+    }
 
     before do
       stub_request(:post, %r{.*api/v2/offers/SAMPLE/shares.*}).
-        with(body: /.*"channel":"sms".*/).
-        to_return(body: '{"ok": true, "result": {"share":{"id":2096}}}')
+        to_return(body: '{
+          "ok": true,
+          "result": {
+            "success": true,
+            "validation_only": false,
+            "stats": {
+              "currently_sent": 2,
+              "currently_not_sent": 0,
+              "previously_sent": 0,
+              "total_sent": 2,
+              "sent_limit_exceeded": false,
+              "left_emails": 20
+            },
+            "recipients": {
+              "friend1@example.com": {
+                "currently_sent": true,
+                "previously_sent": false,
+                "email_valid": true,
+                "self_referral": false,
+                "unsubscribed": false,
+                "blacklisted": false,
+                "meets_criteria": true,
+                "sharable": true
+              },
+              "friend2@example.com": {
+                "currently_sent": true,
+                "previously_sent": false,
+                "email_valid": true,
+                "self_referral": false,
+                "unsubscribed": false,
+                "blacklisted": false,
+                "meets_criteria": true,
+                "sharable": true
+              }
+            },
+            "reward": null
+          }
+        }')
+    end
+
+    it ".direct via email" do
+      expect(Talkable::API::Share.direct(short_url_code,
+        channel: channel,
+        recipients: 'friend1@example.com,friend2@example.com',
+        subject: 'Hello!',
+        body: 'World!',
+        reminder: false)).to eq(response_hash)
     end
 
-    it "success" do
-      expect(Talkable::API::Share.create(short_url_code, channel)).to eq({share: {id: 2096}})
+    it ".direct fails when not email (for the time being)" do
+      expect{Talkable::API::Share.direct(short_url_code,
+        channel: 'random',
+        recipients: 'Nathan',
+        subject: 'Hello!',
+        body: 'World!',
+        reminder: false)}.to raise_error
```

Comment:
I would be more specific here (e.g `raise_error(ArgumentError, /only email sharing channel is allowed/)`